portfolio/src/main/webapp/script.js

```diff
@@ -24,16 +25,20 @@ function loadComments() {
   });
 }
 
+/** UNDER CONSTRUCTION */
+function deleteTask(task) {
+  const params = new URLSearchParams();
+  params.append('id', task.id);
+  fetch('/delete-task', {method: 'POST', body: params});
+}
+
+/** UNDER CONSTRUCTION */
 function deleteComments() {
-  fetch('/delete-data').then(response => response.json()).then((tasks) => {
-    const taskListElement = document.getElementById('greeting-container');
-    //var nc = document.getElementById("num-comments").value;
-    
-    tasks.forEach((task) => {
-      taskListElement.removeChild();
+  tasks.forEach((task) => {
```

Comment:
This doesn't look like it will do what we want it to. Feel free to reach out to Chy or myself if you want some help working through this part of the walkthrough.